Description: The shebang has been removed from lib/sanitize.sh.
 This file should not be executed directly and is not executable.
 Lintian rightfully complains about the shebang.
Author: Alexis Sukrieh <sukria@sukria.net>

---
 lib/sanitize.sh |    1 -
 1 file changed, 1 deletion(-)

--- a/lib/sanitize.sh
+++ b/lib/sanitize.sh
@@ -1,4 +1,3 @@
-#! /usr/bin/env bash
 # Copyright © 2005-2010 Alexis Sukrieh
 #
 # See the AUTHORS file for details.
